aspnet/web-forms/overview/getting-started/getting-started-with-aspnet-45-web-forms/membership-and-administration/samples/sample3.cs

```diff
@@ -41,7 +42,7 @@ internal void AddUserAndRole()
         UserName = "canEditUser@wingtiptoys.com",
         Email = "canEditUser@wingtiptoys.com"
       };
-      IdUserResult = userMgr.Create(appUser, "Pa$$word1");
+      IdUserResult = userMgr.Create(appUser, ConfigurationManager.AppSettings["AppUserPasswordKey"]);
```

Comment:
Does that work in Web Forms?

Reply:
I had tested in a Web Forms app, putting in the web.config and pulling back in through the code this way which worked.  It was problematic trying to test it on the _original_ app sample, it doesn't exist anymore.  There are just some files left it refers to. The original app code example does not exist, instead it points off to samples site, and it is not there.  If there is a more proper way to do this for web forms though let me know.